cpp/ql/src/Security/CWE/CWE-676/PotentiallyDangerousFunction.qhelp

```diff
@@ -5,9 +5,11 @@
 <overview>
 <p>This rule finds calls to functions that are dangerous to
 use. Currently, it checks for calls
-to <code>gmtime</code>. See <strong>Related rules</strong>
+to <code>gets</code> and <code>gmtime</code>. See <strong>Related rules</strong>
 below for rules that identify other dangerous functions.</p>
 
+<p>The <code>gets</code> function is one of the vulnerabilities exploited by the Internet Worm of 1988, one of the first computer worms to spread through the Internet.</p>
+
```

Comment:
That's a fun fact, but we don't blacklist every function that's been exploited by a worm. The reason we blacklist it is that there is no way to use it securely.

Reply:
True (note that I didn't write this, I've just moved existing snippets from one `.qhelp` to another and adapted them a little bit).

I've added another sentence to explain your point.